src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -456,13 +455,11 @@ public async Task Get()
             {
                 await service.SetAsync(testSettingNoLabel);
                 // Test
-                Response<ConfigurationSetting> response = await service.GetAsync(key: testSettingNoLabel.Key, options: default, CancellationToken.None);
+                ResponseTask<ConfigurationSetting> task = service.GetAsync(key: testSettingNoLabel.Key, options: default, CancellationToken.None);
```

Comment:
nit: I would rename the task to `getAsyncTask` to make it clear the task is about retrieving the key.